CROSS_COMMUNITY_COUNCIL.md

```diff
@@ -0,0 +1,9 @@
+# Cross Community Council (C3)
```

Comment:
I don't think we have consensus yet about this governance structure, this might be premature